core/state_transition.go

```diff
@@ -336,9 +339,61 @@ func (st *StateTransition) TransitionDb(refunds bool, gasBailout bool) (*evmtype
 	rules := st.evm.ChainRules()
 	vmConfig := st.evm.Config()
 	isEIP3860 := vmConfig.HasEip3860(rules)
+	accessTuples := make(types2.AccessList, 0)
+	if msg.AccessList() != nil {
+		accessTuples = append(accessTuples, msg.AccessList()...)
+	}
+
+	// set code tx
+	auths := msg.Authorizations()
+	if len(auths) > 0 {
+		var b [33]byte
+		data := bytes.NewBuffer(nil)
+		for _, auth := range auths {
+
+			// 2. chainId check
+			if auth.ChainID != 0 && auth.ChainID != st.evm.ChainRules().ChainID.Uint64() {
+				continue
+				//TODO: should these errors be recorded somewhere? perhaps tracer or logs
```

Comment:
@yperbasis the eip says to skip the authorization in case a check fails. I didn't find a good way to record these failures though. Maybe a new tracer method or log (but log isn't used in state transition or other places).

Reply:
Let's log it with debug level.